cartoframes/client/sql_client.py

```diff
@@ -35,18 +32,18 @@ class SQLClient(object):
             ...
     """
 
-    def __init__(self, credentials, session=None):
+    def __init__(self, credentials):
         self._is_org_user = None
-        self._creds = credentials
-        self._context = self._create_context()
+        self._credentials = credentials
+        self._context = context.create_context(credentials)
 
     def query(self, query, verbose=False):
         """Run a SQL query. It returns a `list` with content of the response.
         If the `verbose` param is True it returns the full SQL response in a `dict`.
         For more information check the `SQL API
         documentation
         <https://carto.com/developers/sql-api/reference/#tag/Single-SQL-Statement>`."""
-        response = self._context.execute_query(query.strip())
+        response = self._context.execute_query(query)
```

Comment:
Why aren't we striping the query here now?

Reply:
Because the strip() is done inside the `execute_query` method now.